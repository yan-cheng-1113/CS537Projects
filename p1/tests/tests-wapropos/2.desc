Keyword with Entries - Should search man pages and print proper list pages containing keyword in correct format and order to stdout and return 0.
