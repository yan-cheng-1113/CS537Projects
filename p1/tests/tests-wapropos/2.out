example (1) - an example program
wapropos (1) - a simplified program to search manual page names and descriptions
wman (1) - a simplified manual program
example (2) - an example program
